docs/source/demos/fraud.ipynb

```diff
@@ -68,7 +68,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "X, y = evalml.demos.load_fraud(n_rows=5000)"
+    "X, y = evalml.demos.load_fraud(n_rows=1000)"
```

Comment:
Does the fraud demo still read OK with this change? I seem to remember us changing this because the numbers in the demo were screwy when the data size was too small. But I could be wrong.

Reply:
@freddyaboulton
Looks like for the [fraud demo ](https://feature-labs-inc-evalml--2430.com.readthedocs.build/en/2430/demos/fraud.html
)- both pipelines score evenly on fraud. AUC is actually higher on the fraud optimized pipeline as well. Not sure if this is a change in behavior but could make this section a little confusing.

Reply:
I think the fraud demo is currently broken which is why @bchen1116 is reworking the fraud objective? It's possible we'll have to increase the size of the data once the fraud objective "works" again but when we get there, we can do what we're doing now for lead scoring and limit the model families to keep the computation fast.

So I think this is the simplest way to not spend a lot of time in this notebook while we fix it. Let me know what you think!

![image](https://user-images.githubusercontent.com/41651716/123280136-871f5700-d4d6-11eb-84b0-4ba091cc57be.png)

Reply:
@freddyaboulton works for me! thanks for explaining 😄

Reply:
Ohhh nvm, lower scores are better for fraud! I'll limit the model families now!

Reply:
Ohh, that screen shot was from stable, not latest. Looks like it is broken on latest/main hehe

https://evalml.alteryx.com/en/latest/demos/fraud.html

![image](https://user-images.githubusercontent.com/41651716/123282860-dcf4fe80-d4d8-11eb-820a-c36272665ee0.png)